Cargo.toml

```diff
@@ -85,14 +85,18 @@ chrono = { version = "0.4.25", default-features = false, features = [
     "now",
 ] }
 criterion = { version = "0.5", features = ["async", "async_tokio"] }
-datafusion = { version = "36.0.0", default-features = false, features = [
+datafusion = { version = "37.0.0", default-features = false, features = [
+    "array_expressions",
+    "regex_expressions",
+] }
+datafusion-common = "37.0"
```

Comment:
Datafusion is on 37.1 https://docs.rs/datafusion/latest/datafusion/

Should we bump to datafusion 37.1?

Reply:
Oh yea, maybe just `37` for simplicity.

Reply:
Actually, probably best to do 37.1 just to avoid issues.  It doesn't change that often.

Reply:
Updated.

Reply:
If we don't have a reason to require `>=37.1`, I feel like it would be nice to keep at `37.0`. Then users can use any `37.X` version.

Reply:
Do you mean `37`?  Or does `37.0` allow `37.1`?

Scanning through https://github.com/apache/datafusion/issues/9904 I don't see any defects that would impact us.